stacks-signer/src/signer.rs

```diff
@@ -46,6 +46,7 @@ use wsts::state_machine::coordinator::{
 };
 use wsts::state_machine::signer::Signer as WSTSSigner;
```

Comment:
Can you rename this `SignerStateMachine` or something else descriptive?  `WSTSSigner` is really ambiguous.

Reply:
Yeah I found that name confusing as well. Will change it with pleasure. Thanks for the suggestion :pray:

Reply:
Updated :heavy_check_mark: